lib/stat_tracker.rb

```diff
@@ -24,6 +28,10 @@ def team_collection
     TeamCollection.new(@team_path)
   end
 
+  def game_team_collection
+     GameTeamCollection.new(@game_team_path)
+  end
```

Comment:
every time this method is called it creates a new GameTeamCollection, can we make this into an instance variable